src/epd7in5b_v2/mod.rs

```diff
@@ -221,13 +223,24 @@ where
     fn set_lut(
         &mut self,
         _spi: &mut SPI,
+        _delay: &mut DELAY,
         _refresh_rate: Option<RefreshLut>,
     ) -> Result<(), SPI::Error> {
         unimplemented!();
     }
 
-    fn is_busy(&self) -> bool {
-        self.interface.is_busy(IS_BUSY_LOW)
+    /// wait
+    fn wait_until_idle(&mut self, spi: &mut SPI, delay: &mut DELAY) -> Result<(), SPI::Error> {
+        // C driver first sends the command
+        self.interface.cmd(spi, Command::GetStatus)?;
+        while self.interface.is_busy(IS_BUSY_LOW) {
+            self.interface.cmd(spi, Command::GetStatus)?;
+            // C driver doesn't wait here but we have to
+            // because be want to be able to give back control if we are in a RT thread
+            delay.delay_ms(10);
```

Comment:
3. Which could be used here as well
```suggestion
            delay.delay_ms(self.delay_ms);
```

Reply:
If you look at the comment, the goal is to give back control to scheduler in case of a RT thread, this means the value needed here depends on the host OS, so it should be under the control of the library user, not the developer.
So, I indeed think that those calls to should use `self.delay_ms` but I also think most customized `wait_until_idle` should disappear in favor of a common one in interface. There are only 2 flavors, one with a call to GetStatus and one without.

Reply:
Anyway, having a default value for users who don't care is also interesting